install_builder/deadline-cloud-for-nuke.xml

```diff
@@ -74,13 +73,36 @@
 		</stringParameter>
 	</parameterList>
     <postInstallationActionList>
-		<fnAddPathEnvironmentVariable>
-			<progressText>Setting NUKE_PATH</progressText>
-			<name>NUKE_PATH</name>
-			<value>${nuke_installdir}</value>
-			<scope>${installscope}</scope>
-			<insertAt>end</insertAt>
-		</fnAddPathEnvironmentVariable>
+        <if>
+            <conditionRuleList>
+                <platformTest type="windows"/>
+            </conditionRuleList>
+            <actionList>
+                <fnAddPathEnvironmentVariable>
+                    <progressText>Setting NUKE_PATH</progressText>
+                    <name>NUKE_PATH</name>
+                    <value>${nuke_installdir};${nuke_installdir}/deadline/nuke_submitter</value>
+                    <scope>${installscope}</scope>
+                    <insertAt>end</insertAt>
+                </fnAddPathEnvironmentVariable>
+            </actionList>
+        </if>
+        <if>
+            <ruleEvaluationLogic>or</ruleEvaluationLogic>
+            <conditionRuleList>
+                <platformTest type="osx"/>
+                <platformTest type="linux"/>
+            </conditionRuleList>
+            <actionList>
+                <fnAddPathEnvironmentVariable>
+                    <progressText>Setting NUKE_PATH</progressText>
+                    <name>NUKE_PATH</name>
+                    <value>${nuke_installdir}:${nuke_installdir}/deadline/nuke_submitter</value>
+                    <scope>${installscope}</scope>
+                    <insertAt>end</insertAt>
+                </fnAddPathEnvironmentVariable>
```

Comment:
How does this work on Macs? Does it add the env var to the bash profile?

Reply:
Does the installer even support MacOS?

Reply:
This is a custom action that we have defined. The bulk of the work uses `setInstallerVariable` from InstallBuilder https://releases.installbuilder.com/installbuilder/docs/installbuilder-userguide/variables.html